backend_addon/{{ cookiecutter.__folder_name }}/README.md

```diff
@@ -19,6 +19,37 @@ And to create the Plone site:
 make create_site
 ```
 
+## Add features using `plonecli` or `bobtemplates.plone`
+
+This package provides markers as strings (`<!-- extra stuff goes here -->`) that are compatible with [`plonecli`](https://github.com/plone/plonecli) and [`bobtemplates.plone`](https://github.com/plone/bobtemplates.plone).
+These markers act as hooks to add all kinds of subtemplates, including behaviors, control panels, upgrade steps, or other subtemplates from `plonecli`.
+
+To use them, you need to use `bobtemplates.plone` version 6.3.4 or later, and run the following command.
```

Comment:
This assumes plonecli is already installed. How should the developer install it? Should these docs be updated to use the `make add` command?

Reply:
Done in https://github.com/plone/cookieplone-templates/pull/63/commits/72a2c0866156ebf3fea7bdb2ecec9cfad8af1764